denizen_scripts/survival/repo-link/MobScripts/SlimeStuff.dsc

```diff
@@ -0,0 +1,45 @@
+armor_degradation:
+  type: world
+  debug: false
+  events:
+    on player damaged by slime:
+      - if <context.damager.is_mythicmob> && <context.damager.MythicMob.level.is[MORE].to[5]>:
+          - determine passively cancelled
+          - choose <util.random.int[1].to[4]>:
+              - case 1
+                - if <player.equipment_map.get[helmet].material.name> == *_helmet|turtle_helmet
```

Comment:
This will return invalid if the map returns that the helmet is not in the map.
Missing a colon at the end of the `if` statement
This isn't quite how `==` logic works; regex doesn't go here. optimally, it's best to avoid regex regardless. you want to use `equipment_map[helmet].material.name.ends_with[stuff]` here.
```suggestion
                - if <player.equipment_map.contains[helmet]> <player.equipment_map.get[helmet].material.name.ends_with[_Helmet]>:
```